deeppavlov/configs/classifiers/insults_kaggle_bert_torch.json

```diff
@@ -19,7 +19,7 @@
     "pipe": [
       {
         "class_name": "torch_bert_preprocessor",
```

Comment:
torch_bert_preprocessor -> torch_transformers_preprocessor

Reply:
maybe a single better than plural?
torch_transformer_preprocessor

Reply:
@IgnatovFedor we need your opinion here

Reply:
When I wrote `transformers` I thought about HuggingFace Transformers, not about NN architecture. Don't know what is better here.

Reply:
torch_transformers_preprocessor, agree with @dilyararimovna